### ISSUE TITLE
Melhorar qualidade dos GIFs

### ISSUE THREAD

**Issue by thiagofelix** [thiagofelix/DicionarioLibras#2]
Esta issue é para corrigir a qualidade ruim de alguns GIFs.

**Comment by henriquelbsouza** [comment]
Não seria legal movermos esta issue pro projeto "um-sinal-por-dia" já que todos os gifs estão indexados lá e não aqui?

**Comment by thiagofelix** [comment]
Na real, acho que talvez mova as gifs pra ca. Ruim apontar p/ lá, a gente pode acabar apagando sem querer.

**Comment by henriquelbsouza** [comment]
Sim, faz sentido... Talvez até um repo separado? Pra caso alguém queira contribuir nesses projetos não precise fazer o download de tantos arquivos.

**Comment by thiagofelix** [comment]
Eu recriei os gifs no outro repo, acho que a gente devia jogar no s3, assim o git fica mais leve tbm
